core/src/main/java/nl/nn/adapterframework/doc/objects/SchemaInfo.java

```diff
@@ -8,10 +8,16 @@
 import lombok.Getter;
 import lombok.Setter;
 
-@Getter
-@Setter
 public class SchemaInfo {
+	@Getter
+	@Setter
 	private Map<String, TreeSet<IbisBean>> groups;
+	
+	@Getter
+	@Setter
 	private Set<IbisBean> ibisBeans;
+	
+	@Getter
+	@Setter
 	private List<IbisMethod> ibisMethods;
```

Comment:
Ik stel voor om het steeds op één regel te doen, dat is rustiger om te lezen:
```
  private @Setter @Getter Map<String, TreeSet<IbisBean>> groups;
  private @Setter @Getter Set<IbisBean> ibisBeans;
  private @Setter @Getter List<IbisMethod> ibisMethods;

```

Reply:
Ah, sorry, my last commit still is not OK. Please wait a minute...

Reply:
Now I really fixed it.